confluent_kafka/src/Producer.c

```diff
@@ -398,6 +398,34 @@ static PyObject *Producer_flush (Handle *self, PyObject *args,
         return cfl_PyInt_FromInt(qlen);
 }
 
+
+static void *Producer_purge (Handle *self, PyObject *args,
```

Comment:
I know the source is messed up tab/space-wise, but please try to use 8-space indents in new code, no tabs.
Thank you!